src/gene_demulti/bcftools/config.vsh.yaml

```diff
@@ -0,0 +1,42 @@
+functionality:
+  name: bcftools
+  namespace: gene_demulti
+  description: Filter the variants called by freebayes or cellSNP
+  authors:
+    - name: Xichen Wu
+  arguments:
+    - name: "--vcf"
+      type: file
+      required: true
+      multiple: true
+      multiple_sep: ","
+      description: VCF files, must have the same sample columns appearing in the same order.
+    - name: "--concat"
+      type: boolean_true
+      description: Concatenate or combine VCFs and sort them.
+    - name: "--filter"
+      type: boolean_true
+      description: "Filter VCFs."
+    - name: "--filter_qual"
+      type: integer
+      default: 30
+      description: "Filter VCFs with specified QUAL threshold."
+    - name: "--output"
+      type: file
+      direction: output
+      description: bcftools output directory
+      example: "bcftools_out/"
+  resources:
+    - type: bash_script
+      path: script.sh
+  test_resources:
+    - type: bash_script
+      path: test.sh
+    - path: ../../../resources_test/demuxafy_test_data
```

Comment:
Could you create a script in `workflows/resources_test_scripts/demuxafy_test_data.sh` that creates the test data for this component?